scripts/start.py

```diff
@@ -37,6 +36,7 @@
 from . import build # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
 from . import common # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
 
+from constants import constants # isort:skip  pylint: disable=wrong-import-position, wrong-import-order
```

Comment:
constants require python_utils and in python_utils we import third_party_libs like certifi. In order to import constants we firstly need to install_third_party_libs (line 34)

Reply:
Thanks for the explanation! :)